src/main/java/com/swervedrivespecialties/swervelib/ctre/Falcon500SteerControllerFactoryBuilder.java

```diff
@@ -122,8 +122,8 @@ public ControllerImplementation create(Falcon500SteerConfiguration<T> steerConfi
                 motor.enableVoltageCompensation(true);
             }
             checkCtreError(motor.configSelectedFeedbackSensor(TalonFXFeedbackDevice.IntegratedSensor, 0, CAN_TIMEOUT_MS), "Failed to set Falcon 500 feedback sensor");
-            motor.setSensorPhase(moduleConfiguration.isSteerInverted());
-            motor.setInverted(TalonFXInvertType.CounterClockwise);
+            motor.setSensorPhase(true);
+            motor.setInverted(moduleConfiguration.isSteerInverted() ? TalonFXInvertType.CounterClockwise : TalonFXInvertType.Clockwise);
```

Comment:
This bit should be confirmed as correct as well. The motors need to be flipped for the Mk4i modules - but it makes sense to me that you'd invert the motor as opposed to inverting the sensor phase. Is there a reason to invert the sensor phase instead of the motor direction?